src/crab/splitdbm/graph_views.hpp

```
@@ -0,0 +1,438 @@
+    class AdjConstIterator final {
+      public:
+        AdjConstIterator(const std::vector<VertId>& _inv, const ItG& _v) : inv(_inv), v(_v) {}
+
+        VertId operator*() const { return inv[*v]; }
+
+        AdjConstIterator& operator++() {
+            ++v;
+            return *this;
+        }
+
+        bool operator!=(const AdjConstIterator& other) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)